pyshacl/cli_rules.py

```diff
@@ -0,0 +1,322 @@
+)
+parser.add_argument(
+    '--abort', 
+    dest='abort', 
+    action='store_true', 
+    default=False, 
+    help='Abort on first invalid data.'
+)
+parser.add_argument(
+    '--allow-info',
+    '--allow-infos',
+    dest='allow_infos',
+    action='store_true',
+    default=False,
+    help='Shapes marked with severity of Info will not cause result to be invalid.',
+)
+parser.add_argument(
+    '-w',
+    '--allow-warning',
+    '--allow-warnings',
+    dest='allow_warnings',
+    action='store_true',
+    default=False,
+    help='Shapes marked with severity of Warning or Info will not cause result to be invalid.',
+)
+parser.add_argument(
+    '--max-depth',
+    dest='max_depth',
+    action='store',
+    nargs='?',
+    type=int,
+    help="The maximum number of SHACL shapes \"deep\" that the validator can go before reaching an \"endpoint\" constraint.",
+)
+parser.add_argument(
+    '-d',
+    '--debug',
+    dest='debug',
+    action='store_true',
+    default=False,
+    help='Output additional verbose runtime messages.',
+)
+parser.add_argument(
+    '--focus',
+    dest='focus',
+    action='store',
+    help='Optional IRIs of focus nodes from the DataGraph, the shapes will validate only these node. Comma-separated list.',
+    nargs="?",
+    default=None,
+)
+parser.add_argument(
+    '--shape',
+    dest='shape',
+    action='store',
+    help='Optional IRIs of a NodeShape or PropertyShape from the SHACL ShapesGraph, only these shapes will be used to validate the DataGraph. Comma-separated list.',
+    nargs="?",
+    default=None,
+)
+parser.add_argument(
+    '-f',
+    '--format',
+    dest='format',
+    action='store',
+    help='Choose an output format. Default is \"trig\" for Datasets and \"turtle\" for Graphs.',
+    default='auto',
+    choices=('auto', 'turtle', 'xml', 'trig', 'json-ld', 'nt', 'n3', 'nquads'),
+)
+parser.add_argument(
+    '-df',
+    '--data-file-format',
+    dest='data_file_format',
+    action='store',
+    help='Explicitly state the RDF File format of the input DataGraph file. Default=\"auto\".',
+    default='auto',
+    choices=('auto', 'turtle', 'xml', 'trig', 'json-ld', 'nt', 'n3', 'nquads'),
+)
+parser.add_argument(
+    '-sf',
+    '--shacl-file-format',
+    dest='shacl_file_format',
+    action='store',
+    help='Explicitly state the RDF File format of the input SHACL file. Default=\"auto\".',
+    default='auto',
+    choices=('auto', 'turtle', 'xml', 'trig', 'json-ld', 'nt', 'n3', 'nquads'),
+)
+parser.add_argument(
+    '-ef',
+    '--ont-file-format',
+    dest='ont_file_format',
+    action='store',
+    help='Explicitly state the RDF File format of the extra ontology file. Default=\"auto\".',
+    default='auto',
+    choices=('auto', 'turtle', 'xml', 'trig', 'json-ld', 'nt', 'n3', 'nquads'),
+)
+parser.add_argument('-V', '--version', action=ShowVersion, help='Show PySHACL version and exit.')
```

Comment:
Any reason this is capital `-V` not lowercase `-v`?

Reply:
Thats so it matches the `-V` param in the regular (validator) pyshacl cli tool. It has always used the capital `-V`. I don't remember the original reason.